roles/satellite-clone/tasks/main.yml

```diff
@@ -147,6 +147,13 @@
   command: satellite-installer --upgrade
   when: rhel_migration
 
+- name: Disassociate capsules with lifecycle environments
```

Comment:
May be it is better to add more info in the task name, something like:

`name: Disassociate capsules with lifecycle environments (to avoid cloned Satellite server talking to live Capsules)`